logicrunner/builtin/initialization.go

```diff
@@ -0,0 +1,46 @@
+//
+// Copyright 2019 Insolar Technologies GmbH
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//	   http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+
+package builtin
+
+import (
+	helloworld "github.com/insolar/insolar/logicrunner/builtin/contract/helloworld"
+	"github.com/pkg/errors"
+
+	XXX_insolar "github.com/insolar/insolar/insolar"
```

Comment:
?

Reply:
it's generated file, I've added prefix `XXX_` to be sure if someone created contract with name `insolar` we will avoid name clash.